CHANGES.rst

```diff
@@ -8,13 +8,54 @@ details, see the commit logs at https://github.com/scikit-build/scikit-build
 Next Release
 ============
 
-We are hard at work on the next generation of scikit-build
-`scikit-build-core <https://github.com/scikit-build/scikit-build-core>`_, which
-has had it's first non-development release. We are also continuing to fix bugs,
-make improvements, and backport changes here.
+We are hard at work on the next generation of scikit-build `scikit-build-core
+<https://github.com/scikit-build/scikit-build-core>`_, which will eventually
+replace the backend here. We are also continuing to fix bugs, make improvements,
+and backport changes here.
 
 .. START-BRIEF-CHANGELOG
 
+Scikit-build 0.17.0
+===================
+
+A lot of bug fixes are present in this release, focusing on Windows, PyPy, and
+cross compiling. We've also improved the compatibility with default setuptools
+behaviors a little, and enabled some things that were previously unavailable,
+like overriding the build type via the cmake argument environment variables.
+We've expanded our CI matrix to include Windows and macOS PyPy and some Fortran
+tests on Linux. This release requires Python 3.7+.
+
+Bug fixes
+---------
+
+* Match setuptools behavior for ``include_package_data`` default. by :user:`vyasr` in :pr:`873`
+* Misc. fixes for F2PY and PythonExtensions modules by :user:`benbovy` in :pr:`495`
+* Provide more useful error if user provides ``CMAKE_INSTALL_PREFIX`` by :user:`vyasr` in :pr:`872`
+* Stop assuming that ``.pyx`` files are in the same directory as ``CMakeLists.txt`` by :user:`vyasr` in :pr:`871`
+* Allow build type overriding by :pr:`902`
+* Detect PyPy library correctly on Windows (#701) in :pr:`904`
+* Include library for FindPython for better Windows cross-compiles in :pr:`913`
+* Use f2py's ``get_include`` if present by in :pr:`877`
```

Comment:
```suggestion
* Use f2py's ``get_include`` if present by user:`henryiii` in :pr:`877`
```

Reply:
I've generally avoided adding core maintainers in the changelog, and only listed external contributions. (For historical reasons, that's how it was done before here - in scikit-build-core, I just take the default output from GH, which includes the users).

Reply:
Thanks for the review. Agreed, let's continue by not mentioning core maintainers.

Background: Since I noticed the wording `by in` for some of the entry, I was not sure of the original intent.

Reply:
Yes, I start with the GitHub generated release notes even here, so I’m manually filtering core maintainer mentions - so I missed deleting “by”.